Source/WebCore/editing/cocoa/HTMLConverter.mm

```diff
@@ -1316,6 +1317,12 @@ static Class _WebMessageDocumentClass()
     if (fileWrapper || usePlaceholder) {
         NSUInteger textLength = [_attrStr length];
         RetainPtr<NSTextAttachment> attachment = adoptNS([[PlatformNSTextAttachment alloc] initWithFileWrapper:fileWrapper.get()]);
+
+        if (auto& ariaLabel = element.getAttribute("aria-label"_s); !ariaLabel.isEmpty())
+            attachment.get().accessibilityLabel = ariaLabel;
```

Comment:
Nit: We generally prefer to use square bracket syntax with `RetainPtr`.

`[attachment setAccessibilityLabel:ariaLabel]`